### PR TITLE
Update docker-assembly with removed files

### PR DESCRIPTION
**Pull request by jotak**

### CODE REVIEW

**Comment by rhqci** [comment]
Can one of the admins verify this patch?

File: docker-dist/src/main/docker/docker-assembly.xml [hunk 1]
```diff
@@ -64,11 +64,6 @@
       <fileMode>755</fileMode>
     </file>
     <file>
-      <source>src/main/resources/check-cnode.sh</source>
```
Comment (jmazzitelli) [inline docker-dist/src/main/docker/docker-assembly.xml]: remove this check-cnode.sh as part of this PR, it isn't used anymore so should be deleted.

Comment (jotak) [reply docker-dist/src/main/docker/docker-assembly.xml]: It's actually already gone, and that's why docker:build failed

**Comment by jmazzitelli** [comment]
while you a fixing the docker image, this should also be cleaned up:

https://github.com/hawkular/hawkular-services/blob/hawkular-1275/docker-dist/src/main/docker/Dockerfile#L33

and related stuff in here (METRICS_TTL env var also should be removed and anything else)

**Pull request merged**